media/js/ccnmtljs/bb_views.js

```diff
@@ -48,6 +48,24 @@ var DeletableItemView = BaseItemView.extend({
     removeItem: function ()
     {   
         this.model.destroy();
+    },
+    
+    confirmDeletion: function ()
+    {
+        if(jQuery(this.el).find('.confirm-del'))
+        {
+        	jQuery(this.el).find('.confirm-del').show();
```

Comment:
Might not actually see much performance difference but I would suggest creating a var for jQuery(this.el).find('.confirm-del') and then use it subsequently. This way you are not calling a (expensive) crawling selector function multiple times. Also, you can define css as an object instead of doing multiple calls. so like var.css({'display':'inline', 'color':'red'})...